packages/wrangler/e2e/dev-remote-bindings.test.ts

```diff
@@ -188,15 +174,30 @@ describe.skipIf(!CLOUDFLARE_ACCOUNT_ID)(
 			);
 
 			// This should only include logs from the user Wrangler session (i.e. a single list of attached bindings, and only one ready message)
-			expect(normalizeOutput(worker.currentOutput)).toMatchInlineSnapshot(`
-			"Your Worker has access to the following bindings:
+			// Logs order are not deterministic, so we match against to possible outputs.
```

Comment:
This approach feels a bit awkward. How about we check that the output contains the string `[wrangler:info] Ready on http://<HOST>:<PORT>\n` and then do a snapshot test on the log with that string removed?